Programm started at: 2025-06-02 23:10:36

Running in version 1.1[2025-06-02 23:11:49]
Git repositories synchronized
	

	NoneType: None


[2025-06-02 23:11:49]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-06-02 23:20:57

Running in version 1.2

[2025-06-02 23:43:00]
Git repositories synchronized
	

	NoneType: None


[2025-06-02 23:43:00]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-06-03 01:41:13

Running in version 1.2

[2025-06-03 14:11:38]
Error 1 at reading or storing sensor data.
	'I2C read failed'

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 191, in read_all
    if self.data_available():
       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 172, in data_available
    value = self.pi.i2c_read_byte_data(self.device, CSS811_STATUS)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 2874, in i2c_read_byte_data
    return _u2i(_pigpio_command(self.sl, _PI_CMD_I2CRB, handle, reg))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 1011, in _u2i
    raise error(error_text(v))
pigpio.error: 'I2C read failed'


